src/main/java/uk/co/ramp/covid/simulation/population/Population.java

```diff
@@ -239,28 +239,71 @@ public void allocatePeople() throws ImpossibleWorkerDistributionException {
         }
     }
 
+    private void createNeighbourGroupDistributions(ProbabilityDistribution<List<Household>> adultDist,
+                                                   ProbabilityDistribution<List<Household>> familyDist,
+                                                   ProbabilityDistribution<List<Household>> pensionerDist) {
+        List<Household> adultGroup = new ArrayList<>();
+        List<Household> familyGroup = new ArrayList<>();
+        List<Household> pensionerGroup = new ArrayList<>();
+
+        for (Household h : households) {
+            switch (h.getNeighbourGroup()) {
+                case ADULT: adultGroup.add(h); break;
+                case FAMILY: familyGroup.add(h); break;
+                case PENSIONER: pensionerGroup.add(h); break;
+            }
+        }
+
+        adultDist.add(0.6, adultGroup);
```

Comment:
We might want these settable in future - to avoid more parameter bloat I currently haven't added parameters unless we need them.

Reply:
I think we want to have the parameters file as the place where we record where we have made choices about things like this.  (And the data pipeline will then have some metadata about where parameters have come from, and how much confidence we have in them.)  It's not just about making things settable.

I know it's a bit bloated, but we did make it easier to add new parameters, and I think we should avoid hard coding anything like this.